src/test/java/io/github/programminglife2016/pl1_2016/parser/SegmentCollectionTest.java

```diff
@@ -0,0 +1,75 @@
+package io.github.programminglife2016.pl1_2016.parser;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.*;
+
+/**
```

Comment:
You have some magic numbers in this test class. Also org.junit.Assert.* should be avoided